sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_models.hpp

```diff
@@ -292,6 +298,69 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
     AZ_SECURITY_KEYVAULT_KEYS_DLLEXPORT static const KeyCurveName P521;
   };
 
+  /**
+   * @brief The encryption algorithm to use to protected the exported key material.
+   *
+   */
+  class KeyEncryptionAlgorithm final {
```

Comment:
Seems like you can use the Extensible class from Azure Core here:https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core/inc/azure/core/internal/extendable_enumeration.hpp#L37

Reply:
Done